internal/js-dependency-scanner/target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
```

Comment:
Exclude Maven generated files in the `.gitignore`, see https://github.com/wso2/carbon-ui-server/blob/master/.gitignore#L183-L187

Reply:
Resolved.